frontend/src/locales/en.json

```diff
@@ -1,355 +1,358 @@
-    "Taker fee":"Taker fee",
-    "Number of public BUY orders":"Number of public BUY orders",
-    "Number of public SELL orders":"Number of public SELL orders",
-    
+    "BOTTOM BAR AND MISC - BottomBar.js": "Bottom Bar user profile and miscellaneous dialogs",
+    "Stats For Nerds": "Stats For Nerds",
+    "LND version": "LND version",
+    "Currently running commit hash": "Currently running commit hash",
+    "24h contracted volume": "24h contracted volume",
+    "Lifetime contracted volume": "Lifetime contracted volume",
+    "Made with": "Made with",
+    "and": "and",
+    "... somewhere on Earth!": "... somewhere on Earth!",
+    "Community": "Community",
+    "Support is only offered via public channels. Join our Telegram community if you have questions or want to hang out with other cool Robots. Please, use our GitHub Issues if you find a bug or want to see new features!": "Support is only offered via public channels. Join our Telegram community if you have questions or want to hang out with other cool Robots. Please, use our GitHub Issues if you find a bug or want to see new features!",
+    "Join the RoboSats group": "Join the RoboSats group",
+    "Telegram (English / Main)": "Telegram (English / Main)",
+    "RoboSats Telegram Communities": "RoboSats Telegram Communities",
+    "Join RoboSats Spanish speaking community!": "Join RoboSats Spanish speaking community!",
+    "Join RoboSats Russian speaking community!": "Join RoboSats Russian speaking community!",
+    "Join RoboSats Chinese speaking community!": "Join RoboSats Chinese speaking community!",
+    "Join RoboSats English speaking community!": "Join RoboSats English speaking community!",
+    "Tell us about a new feature or a bug": "Tell us about a new feature or a bug",
+    "Github Issues - The Robotic Satoshis Open Source Project": "Github Issues - The Robotic Satoshis Open Source Project",
+    "Your Profile": "Your Profile",
+    "Your Robot": "Your Robot",
+    "Go to active order #{{orderID}}": "Go to active order #{{orderID}}",
+    "Your current order": "Your current order",
+    "No active orders": "No active orders",
+    "Your token (will not remain here)": "Your token (will not remain here)",
+    "Back it up!": "Back it up!",
+    "Cannot remember": "Cannot remember",
+    "Rewards and compensations": "Rewards and compensations",
+    "Share to earn 100 Sats per trade": "Share to earn 100 Sats per trade",
+    "Your referral link": "Your referral link",
+    "Your earned rewards": "Your earned rewards",
+    "Claim": "Claim",
+    "Invoice for {{amountSats}} Sats": "Invoice for {{amountSats}} Sats",
+    "Submit": "Submit",
+    "There it goes, thank you! 🥇": "There it goes, thank you! 🥇",
+    "You have an active order": "You have an active order",
+    "You can claim Sats!": "You can claim Sats!",
+    "Public Buy Orders": "Public Buy Orders",
+    "Public Sell Orders": "Public Sell Orders",
+    "Today active Robots": "Today active Robots",
```

Comment:
There are two of these because they print in two different places: the bottom bar and the Exchange Summary. In the bottom bar, all words are capitalized. In the Exchange summary, they are not capitalized. These are different keys.

Reply:
```
135    "Public Buy Orders"
136    "Public Sell Orders"
137    "Today Active Robots"

143    "Public buy orders"
144    "Public sell orders"
146    "Today active robots"
```

Following this: https://github.com/Reckless-Satoshi/robosats/pull/97#discussion_r851774610

> Sounds good to me! Then we can remove the capitalization of every word in the helper texts of Maker Page (es.json is also capitalized in the same sentences)

This way looks good for me:

![image](https://user-images.githubusercontent.com/68381662/163727962-f295549e-5d25-45d1-adf4-b1aba254a47b.png)

Reply:
> "Public" adjective is needed?

Indeed, not needed.